### PR TITLE
added a docker-compose file: swarm_consul.yml. 

### PR DESCRIPTION
**Pull request by edagan**
This simulates our customer's environment.
Please review.

### CODE REVIEW

File: swarm_consul.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+     - /DATA/postgres/pgdata:/var/lib/postgresql/data/pgdata
+    ports:
+     - "5432:5432"
+
+  clair:
+    env_file: .env
+    image: quay.io/coreos/clair:v1.1.0
+    volumes:
+      - ./clair_config:/config:ro
+      - /tmp/clair-data:/tmp
```
Comment (rqc) [inline swarm_consul.yml]: map to `/DATA`

File: swarm_consul.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+     - /DATA/postgres/pgdata:/var/lib/postgresql/data/pgdata
+    ports:
+     - "5432:5432"
+
+  clair:
+    env_file: .env
```
Comment (rqc) [inline swarm_consul.yml]: remove `.env`

File: swarm_consul.yml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      - /tmp/clair-data:/tmp
+    entrypoint: /bin/bash
+    command: -c "sleep 30 && clair --config=/config/config.yaml"
+    ports:
+     - "6060:6060"
+     - "6061:6061"
+    depends_on:
+      - postgres
+
+  rethinkdb:
```
Comment (rqc) [inline swarm_consul.yml]: `rethinkdb` container should have db persisted and mounted to the host (under `/DATA`)

**Comment by edagan** [comment]
Will fix and open a new PR.
Thanks!

**Pull request closed without merging**